api/frontend.ts

```diff
@@ -18,11 +19,18 @@ export function rootRedirect (req, res, next) {
 
 export function uiConfig (req, res) {
   res.header('content-type', 'application/javascript')
-  res.write(`
+
+  if (env.AUTH_CONFIG_FILE) {
+    const stream = fs.createReadStream(env.AUTH_CONFIG_FILE)
+    stream.pipe(res)
```

Comment:
How are we actually going to use this? You would have multiple configuration files packaged in the docker image and choose the right one by changing the value of `AUTH_CONFIG_FILE`?

I see limited utility of this change 🤔

Reply:
No, the plan is to have this file injected at runtime with either configmaps in k8s or read-only volumes in docker

e.g. with docker-compose

```yaml
version: 3.4
services:
  data-cube-curation:
    image: zazuko/datacube:latest

    environment:
      NODE_ENV: production
      AUTH_CONFIG_FILE: "/config/auth.js"

    volumes:
      - "./datacube/:/config:ro"

Reply:
Ah, volumes, of course 🤦